static/js/tools/stat_var/page.tsx

```diff
@@ -44,7 +53,9 @@ class Page extends Component<unknown, PageStateType> {
     this.state = {
       description: "",
       displayName: "",
+      entities: [],
```

Comment:
since you have both "entities" and "sources", should entities be places here?

Reply:
"entities" refers to the source/dataset that the user selects, while "sources" (updated to sourceMap) refers to the options in the autocomplete - hope this makes more sense, but I can change if not!